pumps/hybrid.go

```diff
@@ -38,8 +38,9 @@ var (
 
 // HybridPump allows to send analytics to MDCB over RPC
 type HybridPump struct {
-	aggregated    bool
-	trackAllPaths bool
+	aggregated             bool
+	trackAllPaths          bool
+	storeAnalyticPerMinute bool
```

Comment:
What this value is for?

Reply:
If this is enabled, analytics will be stored per minute. Otherwise, analytics will be stored per hour, which is current behaviour